github/issues.go

```diff
@@ -28,6 +28,7 @@ type Issue struct {
 	ID                *int64            `json:"id,omitempty"`
 	Number            *int              `json:"number,omitempty"`
 	State             *string           `json:"state,omitempty"`
+	StateReason       *string           `json:"state_reason,omitempty"`
```

Comment:
Since the options are few, can we please add a comment to this field?
(You will probably also have to reformat this file if you take this suggestion directly as-is.)

```suggestion
	// StateReason can be one of: "completed", "not_planned", "reopened".
	StateReason       *string           `json:"state_reason,omitempty"`
```